docs/wikidocs.rst

```diff
@@ -1,19 +0,0 @@
-.. _wiki-chapter:
-
-========================
-Important Wiki Documents
-========================
```

Comment:
Why are we dropping this?

Reply:
The doc points to (and is about) https://support.mozilla.org/en-US/kb/managing-landing-page-content, none of which applies any more (from what I can see).